.github/workflows/releaseutils.yml

```diff
@@ -0,0 +1,30 @@
+name: Add sbom
+
+on:
+  workflow_dispatch:
+  workflow_run:
+        workflows: [test-latest-release]
+        types: [completed]
+
+permissions: read-all
+
+jobs:
+    add-sbom:
+        name: Add SBOM
+        runs-on: ubuntu-latest
+        if: ${{github.event.workflow_run.conclusion=='success'}}
+        steps:
+            - name: checkout code
+              uses: actions/checkout@v4
+
+            - name: Create SBOM
+              uses: advanced-security/sbom-generator-action@v0.0.1
+              id: sbom
+              env: 
+                  GITHUB_TOKEN: ${{ github.token }}

```

Comment:
You are replying to a confused comment of mine that I have since deleted after I found in https://docs.github.com/en/actions/security-for-github-actions/security-guides/automatic-token-authentication#about-the-github_token-secret the statement that "The token is also available in the github.token context".

Reply:
fine ,probably a delay. I'll redo this workflow and ping you once its done.